### ISSUE TITLE
Error: No entry or class found for 'Tests.now'

### ISSUE THREAD

**Issue by mpaolo** [matomo-org/matomo#12989]
hi

just updated to 3.5.1-b2. Now dashboard yields that error box in RT visitors count and visitors in RT widgets instead of the usual graphs

**Comment by mpaolo** [comment]
ok thanx, guess a new beta update will be released soon then

**Comment by Findus23** [comment]
@mpaolo The stable version has already been released.

**Comment by mpaolo** [comment]
ok but I'm on latest update 3.5.1-b2 and did not get any update prompt so far, to the version with the fix - what am I supposed to do to get the fixed release? (re)set release channel to 'latest stable' instead of 'latest beta' in general settings?

**Comment by Findus23** [comment]
Hi, the update check is cached for a few hours. But you can manually recheck by clicking on the update button on the admin page.

**Comment by mpaolo** [comment]
yep actually right after sending last comment and getting back from 'settings' got the prompt, installed and all fine now :)
sorry for the noise
